sys/net/gnrc/network_layer/sixlowpan/frag/sfr/gnrc_sixlowpan_frag_sfr.c

```diff
@@ -1481,8 +1481,12 @@ static void _sched_next_frame(void)
 {
     if (CONFIG_GNRC_SIXLOWPAN_SFR_INTER_FRAME_GAP_US > 0) {
         int state = irq_disable();  /* make timer check atomic */
+#if !IS_USED(MODULE_ZTIMER_XTIMER_COMPAT)
```

Comment:
hm, maybe it makes more sense to just provide an `xtimer_is_set(const xtimer_t *xtimer) {xtimer->offset || xtimer->long_offset)` and use it here. I'll make a PR.

Reply:
https://github.com/RIOT-OS/RIOT/pull/17630